### PR TITLE
feat : Unsplash API & edit functionality

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR :
- I'm adding Unsplash API for background images 
(on every refresh new image. Don't refresh too many times as the API has a limit of 50request/hr)
- Adding edit functionality for name and the main focus
- Changing the manifest.json to make it an extension
- Updating `readme.md`

The edit buttons are not visible on the preview due to the Netlify status bar. 
Click on the bottom rightmost arrow to shrink the status bar and the edit buttons will be visible then

### CODE REVIEW

**Review by iRohitGaur — APPROVED** [review]
Clean and understandable code.

Good to merge 👍🏻

File: src/pages/css/mainApp.css [hunk 1]
```diff
@@ -21,3 +21,28 @@
...
+
+.btn {
+  position: fixed;
+  bottom: 10px;
+  background-color: transparent;
+  padding: 5px;
+  border-radius: 5px;
+  border: none;
+  cursor: pointer;
+  color: #ffff;
```
Comment (parulgupta2001) [inline src/pages/css/mainApp.css]: you have to use var for color

**Review by parulgupta2001 — APPROVED** [review]
Apply the suggestions otherwise code is clean

File: src/pages/css/mainApp.css (hunk 1 above)
Comment (sudipkundu999) [reply src/pages/css/mainApp.css]: Done

**Pull request merged**